packages/jetkit-cdk/src/test/sampleApp.ts

```diff
@@ -0,0 +1,69 @@
+import { HttpMethod } from "@aws-cdk/aws-apigatewayv2";
+import { APIGatewayProxyHandlerV2 } from "aws-lambda";
+import { BaseModel } from "../database/baseModel";
+import { Column, Entity } from "typeorm";
+import { APIEvent } from "../api/base";
+import { CrudApiView } from "../api/crud/base";
+import { CrudApi, Route, SubRoute } from "../registry";
+
+// sample database model
+@Entity()
+export class Album extends BaseModel {
+  @Column({ nullable: true })
+  title: string;
+}
+
+// sample CRUD view
+@CrudApi({
+  model: Album,
+  path: "/album",
+  memorySize: 512,
+  environment: {
+    LOG_LEVEL: "DEBUG",
+  },
```

Comment:
Will this set the log level only for Lambdas generated by this decorator?

Would be cool to have a config in 1 place and just use it everywhere like we do with Python.

Maybe with this https://www.npmjs.com/package/convict ?

Reply:
> Will this set the log level only for Lambdas generated by this decorator?
Yeah, it's just a demo. We'll need some way to set top-level config defaults


> Would be cool to have a config in 1 place and just use it everywhere like we do with Python.
> Maybe with this https://www.npmjs.com/package/convict ?
Sounds great to me! I was imagining a very very thin "app" that jetkit/cdk provides that has config, logging, maybe some plugin architecture but not much else.